packages/contracts/src/contracts/current/protocol/AssetTransferProxy/IAssetTransferProxy.sol

```diff
@@ -0,0 +1,80 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+
+import "./IAssetProxy.sol";
+import "../../utils/Authorizable/IAuthorizable.sol";
+
+contract IAssetTransferProxy is
+    IAuthorizable
```

Comment:
`is IAssetProxy`

I'd also rename this to IAssetProxyDispatcher`.

The cool thing is that the Dispatcher is itself a valid `IAssetProxy` so we could nest them.

Reply:
I really like this idea +1